fmriprep/workflows/bold/confounds.py

```diff
@@ -687,6 +690,13 @@ def _getusans_func(image, thresh):
     ica_aroma_confound_extraction = pe.Node(ICAConfounds(err_on_aroma_warn=err_on_aroma_warn),
                                             name='ica_aroma_confound_extraction')
 
+    ica_aroma_metadata_fmt = pe.Node(
+        TSV2JSON(index_column='IC', output=None, enforce_case=True,
+                 additional_metadata={'Method': {
+                                      'Name': 'ICA-AROMA',
+                                      'Version': '0.3-beta'}}),
```

Comment:
Just to note that I've been looking into how to query ICA-AROMA's version, and it seems that hard-coding it is the only way. Please note that this metadata may not be true if the user has a bare-metal installation. Any thoughts @effigies? Perhaps remove it?

Reply:
Okay, actually we are using v0.4.4-beta, as per Dockerfile.

Reply:
Would it make sense just to help package ICA-AROMA, make sure it has `__version__`, and put it on PyPI?